finagle/h2/src/main/scala/io/netty/handler/codec/http2/H2FrameCodec.scala

```diff
@@ -210,7 +210,6 @@ object H2FrameCodec {
       val fr = new Http2InboundFrameLogger(new DefaultHttp2FrameReader, frameLogger)
       new DefaultHttp2ConnectionDecoder(conn, encoder, fr)
     }
-    decoder.localSettings(settings)
```

Comment:
Actually, I may be wrong about that. Would be best to setup a simple test -- i usually run `nghttpd --no-tls` and have linkerd connect to it.  nghttpd will dump all of the settings received, which will be helpful.  The `nghttp` client can be used to initiate an h2 request through linkerd, and will dump any settings received.